### DISCUSSION TITLE
关于ppocr_v4模型

### DISCUSSION THREAD

**Discussion by HHaidai** [PaddlePaddle/PaddleOCR#13964] [Q&A]
官方开源的几个推理模型中，v4-server我跑出来是效果最好的，但是识别的速度相比于其他模型慢上非常多，有什么办法可以提高其速度么？在模型微调中，我看官方推荐的预训练模型是v3版本的蒸馏模型，请问v4版本有没有类似的预训练模型和文档？

**Comment by GreatV** [comment]
可以先按照这个教程来，换成ppocrv4的相关模型应该就可以，主要是文档没有更新 https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_compress/knowledge_distillation.html#11
